src/libs/Navigation/Navigation.js

```diff
@@ -74,7 +88,7 @@ function openDrawer() {
  * @private
  */
 function closeDrawer() {
-    if (!canNavigate('closeDrawer')) {
+    if (!canNavigate('openDrawer')) {
```

Comment:
Should this be `closeDrawer`?

```suggestion
    if (!canNavigate('closeDrawer')) {
```

Reply:
Yes, was just rolling back a lot of changes so missed this change.